### PR TITLE
Disable text selection in timeline and credits

### PR DESCRIPTION
**Pull request by jdfwarrior**
This has bugged me for a long time. Pressing Ctrl + A to select all text selects the text in the timeline and the credits area. Maybe you actually want the credits area? But it seems like the timeline shouldn't be selectable because, once you get it selected, you have to click very specific areas to deselect and it's annoying.

Text in the animation controller can still be selected right now. I may try to figure out how to resolve that as well.

No changes to the changelog necessary (in my opinion) since it's something some people may have never noticed in the first place and doesn't modify the API at all.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thank you so much for the pull request @jdfwarrior! I noticed this is your first pull request and I wanted to say welcome to the Cesium community!

The [Pull Request Guidelines](https://github.com/CesiumGS/cesium/blob/main/CONTRIBUTING.md#pull-request-guidelines) is a handy reference for making sure your PR gets accepted quickly, so make sure to skim that.

* :x: Missing `CONTRIBUTORS.md` entry.
    * Please add yourself to the [contributors](https://github.com/jdfwarrior/cesium/blob/disable-text-selection/CONTRIBUTORS.md) file!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/disable-text-selection/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

File: Source/Widgets/CesiumWidget/CesiumWidget.css [hunk 1]
```diff
@@ -18,6 +18,7 @@
   font-size: 10px;
   text-shadow: 0px 0px 2px #000000;
   padding-right: 5px;
+  user-select: none;
```
Comment (ggetz) [inline Source/Widgets/CesiumWidget/CesiumWidget.css:21]: I think its arguable that the credits shouldn't be selectable. It's [possible for users to add additional credits that show onscreen](https://cesium.com/learn/cesiumjs/ref-doc/Credit.html).

File: Source/Widgets/Timeline/Timeline.css [hunk 2]
```diff
@@ -4,6 +4,7 @@
   bottom: 0;
   overflow: hidden;
   border: solid 1px #888;
+  user-select: none;
```
Comment (ggetz) [inline Source/Widgets/Timeline/Timeline.css:7]: It looks like anywhere else in the css where we apply this rule, we apply all the browser-specific rules as well. Let's do so here for consistency. 

```
  -moz-user-select: none;
  -webkit-user-select: none;
  -ms-user-select: none;
  user-select: none;
```

**Review by ggetz — COMMENTED** [review]
Thanks @jdfwarrior! I have a few comments. 

I would also ask that you add a line to `CHANGES.md`. Although the change will most likely not be noticed by most, it will keep a record of the change in case it affects someone.

File: Source/Widgets/CesiumWidget/CesiumWidget.css (hunk 1 above)
Comment (jdfwarrior) [reply Source/Widgets/CesiumWidget/CesiumWidget.css]: Fair enough. Yeah I wasn't 100% sure that this was needed for the credits.

File: Source/Widgets/Timeline/Timeline.css (hunk 2 above)
Comment (jdfwarrior) [reply Source/Widgets/Timeline/Timeline.css:10]: Added that in. Thanks @ggetz

**Comment by ggetz** [comment]
Thanks @jdfwarrior, and congrats on your first cesium PR!

**Pull request merged**